static/.eslintrc

```diff
@@ -87,12 +108,24 @@ Do not edit this file in your local project
     "no-label-var": 2,
     "no-labels": [2, { "allowLoop": false, "allowSwitch": false }],
     "no-lone-blocks": 2,
+    "no-lonely-if": 2,
```

Comment:
http://eslint.org/docs/rules/no-lonely-if#rule-details

I'm using the second invalid example quite often
```js
if (condition) {
    // ...
} else {
    if (anotherCondition) {
        // ...
    } else {
        // ...
    }
}
```

Do you think that's hard to read?

and that would be valid.
```js
if (condition) {
    // ...
} else {
    if (anotherCondition) {
        // ...
    }
    doSomething();
}
```

IMO This rule can cause confusion if somebody needs to modify some existing code.